jenkins-pipelines/oss/longevity/longevity-100gb-splitted-networks-4h.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    region: 'eu-west-1',
+    availability_zone: 'c',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: '''["test-cases/longevity/longevity-100gb-4h.yaml", "configurations/network_config/scylla_addresses_on_different_interfaces.yaml"]''',
```

Comment:
@fruch what about this one? Do you want to run regular 100g with this configuration? Or keep this new pipeline?

Reply:
The short one we can leave for experiments, and investigations